sql/catalyst/src/main/java/org/apache/spark/sql/catalyst/expressions/UnsafeRow.java

```diff
@@ -230,27 +237,32 @@ public int size() {
   }
 
   @Override
-  public StructType schema() {
-    return schema;
-  }
-
-  @Override
   public Object get(int i) {
```

Comment:
Similar to my comments on the other methods, do you mind adding a comment to `get()` to explain which types we expected it to be called on?